scripts/ci-entrypoint.sh

```diff
@@ -107,19 +122,11 @@ run_upstream_e2e_tests() {
         fi
     fi
 
-    # get the number of worker nodes
-    NUM_NODES="$(kubectl get nodes --kubeconfig="$KUBECONFIG" \
-    -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.taints}{"\n"}{end}' \
-    | grep -cv "node-role.kubernetes.io/master" )"
-
-    # wait for all the nodes to be ready
-    kubectl wait --for=condition=Ready node --kubeconfig="$KUBECONFIG" --all || true
-
     # setting this env prevents ginkg e2e from trying to run provider setup
     export KUBERNETES_CONFORMANCE_TEST="y"
     # run the tests
     (cd "$(go env GOPATH)/src/k8s.io/kubernetes" && ./hack/ginkgo-e2e.sh \
-    '--provider=skeleton' "--num-nodes=${NUM_NODES}" \
```

Comment:
`--num-nodes` parameter isn't strictly required when running upstream E2E tests since the test framework will auto-detect how many nodes there are if this flag is not provided.